pkg/docker/config/config.go

```diff
@@ -51,6 +51,14 @@ func SetAuthentication(sys *types.SystemContext, registry, username, password st
 	})
 }
 
+func additionalAuthFiles(sys *types.SystemContext) []string {
```

Comment:
I don’t quite think there four lines are with a function… it’s three lines in the caller, which already has a `paths` variable, and we have ~5 lines of overhead with defining the function.

Rather, the full computation of all candidate paths should probably be unified between `GetAuthentication` and `GetUserLoggedIn`; it seems a bit unlikely that the existing divergence was intentional (OTOH I haven’t checked the past discussions.)

Reply:
> I don’t quite think there four lines are with a function

“are worth a function”… macOS autocorrection is infuriating sometimes.